src/MessageProperty/WireEncryptedStringConversions.cs

```diff
@@ -11,6 +11,17 @@ public static void EncryptValue(this IEncryptionService encryptionService, WireE
             wireEncryptedString.Value = null;
         }
 
```

Comment:
I don't think there is anything necessary. It will yield build errors when we try to update the core dependency to v7 which will force us to remove this code and we're good.